src/pages/events/createEvents/createEventCriteria/CriteriasPaper.tsx

```diff
@@ -0,0 +1,210 @@
+import React, {useContext, useEffect, useState} from 'react'
+import useCreateEventStyles from '../../../../assets/styleJs/events/createEvent'
+import {
+  Box,
+  Button,
+  CircularProgress,
+  TextField,
+  Typography,
+} from '@mui/material'
+import {
+  ICriteriaPaperInfo,
+  IEvaluatee,
+  IratingScoreObj,
+} from '../TypesEvents'
+import {v4 as uuidv4} from 'uuid'
+import {createEventReducerTypes} from '../../../../types/createEventTypes'
+import {EventContext} from '../EventsContext'
+import axiosData from '../../../../axiosData'
+import {AxiosResponse} from 'axios'
+import Dialog from '@mui/material/Dialog'
+import DialogActions from '@mui/material/DialogActions'
+import CriteriaCard from "./CriteriaCard";
+import RatingScore from "./RatingScore";
+import BonusPercentage from "./BonusPercentage";
+
+const CriteriasPapers = () => {
+  const UseEventContext = () => useContext(EventContext)
+  const {state, dispatchContext} = UseEventContext()
+  const classes = useCreateEventStyles()
+
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [criteriaData, setCriteriaData] = useState<ICriteriaPaperInfo[]>([])
+  const [evaluateesList, setEvaluateesList] = useState<IEvaluatee[]>([])
+
+  const [ratingFinalValue, setRatingFinalValue] = useState<IratingScoreObj>({
+    from: 1,
+    to: 10,
+    id: Math.random(),
+    isSelected: false
+  })
+
+  const [percentageIsSet, setPercentageIsSet] = useState<boolean>(false)
+  const [ratingScoreArr, setRatingScoreArr] = useState<IratingScoreObj[]>([])
+
+  useEffect(() => {
+    setEvaluateesList(state.evaluatees)
+    const getCriterias = async () => {
+      try {
+        setIsLoading(true)
+        const response: AxiosResponse = await axiosData.get('/criteria')
+        await setCriteriaData(response.data)
+        setIsLoading(false)
+      } catch (err) {
+        setIsLoading(false)
+      }
+    }
+    const getRatings = async () => {
+      try {
+        setIsLoading(true)
+        const response: AxiosResponse = await axiosData.get('/rating')
+        await setRatingScoreArr(response.data)
+        setIsLoading(false)
+      } catch (err) {
+        setIsLoading(false)
+      }
+    }
```

Comment:
Use 1 loading state for both, and 1 try catch

Reply:
done